solang-parser/src/lexer.rs

```diff
@@ -353,6 +353,7 @@ pub enum LexicalError {
     DoublePoints(Loc),
     UnrecognisedDecimal(Loc),
     ExpectedFrom(Loc, String),
+    ExponantTooLarge(Loc, String),
```

Comment:
Isn't it `Exponent` instead of `Exponant`?

Reply:
Yes, thanks for catching that